dspace-api/src/main/java/org/dspace/eperson/dao/impl/EPersonDAOImpl.java

```diff
@@ -96,14 +95,14 @@ public int searchResultCount(Context context, String query, List<MetadataField>
                                                       .toLowerCase() + " FROM EPerson as " + EPerson.class
             .getSimpleName().toLowerCase();
 
-        List<MetadataField> sortFields = ListUtils.EMPTY_LIST;
+        List<MetadataField> sortFields = Collections.EMPTY_LIST;
 
-        if (metadataSortField != null) {
+        if (metadataSortField != null) {ItemRestRepositoryIT
```

Comment:
Here's the compilation failure.  A random `ItemRestRepositoryIT` was accidentally included here.